docs/RELEASE-NOTES.rst

```diff
@@ -13,6 +13,7 @@ Added Functionality
 Bug Fixes
 `````````
 * Adds correct Marathon healthCheck path to BIG-IP Health Monitor Send String  `[#256] <https://github.com/F5Networks/marathon-bigip-ctlr/issues/256>`_
+* Corrected a comparison problem in CCCL that caused unnecessary updates for BIG-IP Virtual Server resources.
```

Comment:
Please look at how we are referencing issues in the Bug Fixes section under 1.4.0 for https://github.com/F5Networks/k8s-bigip-ctlr/blob/master/docs/RELEASE-NOTES.rst. It would be nice to have this be consistent with those.

Reply:
I see those in k8s 1.4, but those 'issue' links aren't working, so maybe that's not the right way either.

Reply:
@jputrino - can you please look at why this isn't working for @edarzins ?

Reply:
The format should be 
```
:issues:`256`
```

Now that my PR has been merged, this should work. Note that it only works in the HTML version, not in GitHub.